testing/src/uploadResultTQ.js

```diff
@@ -0,0 +1,58 @@
+var request = require('request');
+var fs = require('fs');
+import * as dotenv from "dotenv";
+
+
```

Comment:
Please ensure Prettier is installed and formatting the code when saving (configurable on VSCode). These blank lines would be automatically removed, for instance, if it was enabled.